commons-numbers-complex-arrays/src/test/java/org/apache/commons/numbers/complex/arrays/ComplexListTest.java

```diff
@@ -130,14 +131,12 @@ void testSetAddAndAddAllNullPointerException() {
         Assertions.assertThrows(NullPointerException.class, () -> list1.add(0, null));
         Assertions.assertThrows(NullPointerException.class, () -> list1.set(1, null));
 
+        ComplexList copy = ComplexList.interleaved();
```

Comment:
Create this copy at the start of the method. When used at the end of the method this will also check the add methods do not result in modifications of the list.